Source/ZoomNet/Utilities/DiagnosticInfo.cs

```diff
@@ -117,7 +117,10 @@ public object[] GetLoggingParameters()
 
 			if (request != null)
 			{
-				logParams.AddRange([request.Method.Method, request.RequestUri, request.Version]);
+				logParams.Add(request.Method.Method);
```

Comment:
Are we changing this due to Language Version Compatibility?

Reply:
Yeap, the original repository is using .net9 and 12 language version.